src/SequencesProcessor.ts

```diff
@@ -67,7 +67,10 @@ export const setSequenceValue = async (
 
     const seqName: string = getSequenceName(tableName, columnName);
     const sql = `SELECT SETVAL(\'"${conversion._schema}"."${seqName}"\', 
-                (SELECT MAX("${columnName}") FROM "${conversion._schema}"."${tableName}"));`;
+                    GREATEST(
```

Comment:
@jhere1 
In general - good catch!
The question is, shouldn't we set sequence value to zero if table is empty?

Reply:
Yeah, the nextval would have been 2.  Setting sequence value to 0 would involve changing the sequence definition min value.  Take a look at my latest commit.  Think this is cleaner.  it does what we want (have nextval be 1) by saying the value of 1 is not used yet.